<launch>
 <node pkg="hw7pkg" name="my_node" type="cvnode.py"/>
 <node pkg="image_processing_hw" name="puball" type="image_pub_all.py"/>
 <node pkg="rqt_image_view" type="rqt_image_view" name="image_view_cropped" args="/image_cropped"/>
 <node pkg="rqt_image_view" type="rqt_image_view" name="image_view_white" args="/image_white"/>
 <node pkg="rqt_image_view" type="rqt_image_view" name="image_view_yellow" args="/image_yellow"/>
</launch>
